### PR TITLE
docs: add fluid, vertical video example

### PR DESCRIPTION
**Pull request by RyKilleen**
## Description
Creates an example in the sandbox for fluid videos, specifically a vertical video example.

## Specific Changes proposed
Please list the specific changes involved in this pull request.

## Requirements Checklist
- [ ] Feature implemented / Bug fixed
- [X ] If necessary, more likely in a feature request than a bug fix
  - [ ] Change has been verified in an actual browser (Chome, Firefox, IE)
  - [ ] Unit Tests updated or fixed
  - [ ] Docs/guides updated
  - [X] Example created ([starter template on JSBin](https://codepen.io/gkatsev/pen/GwZegv?editors=1000#0))
- [ ] Reviewed by Two Core Contributors

### CODE REVIEW

File: sandbox/vertical-video.html.example [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        width: calc(100vh * 0.5625); 
+        max-width: 100%;
+        margin: 0 auto;">
+  <video-js
+    id="vid1"
+    controls
+    preload="auto"
+    class="vjs-fluid"
+    >
+    <source src="http://www.exit109.com/~dnn/clips/RW20seconds_1.mp4" type="video/mp4">
```
Comment (gkatsev) [inline sandbox/vertical-video.html.example:29]: is this a source we can rely on?

Comment (RyKilleen) [reply sandbox/vertical-video.html.example:29]: That's a valid question, I'm open to any other public test videos! I included based solely on this page here:

http://www.exit109.com/~dnn/clips/VerticalVideo.html

Comment (gkatsev) [reply sandbox/vertical-video.html.example:29]: Thanks, it sounds like it's up for usage, though, we may want to rehost the file.
When I look the other day, that page wasn't loading for me.

File: sandbox/vertical-video.html.example [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    <pre>open http://localhost:9999/sandbox/vertical-video.html</pre>
+  </div>
+
+  <!--  To make this truly responsive, we have to constrain the container's width.        -->
+  <!--  Because vertical videos are concerned more with height, we base it off vh units.  -->
+  <!-- calc(100vh * 0.5625) is equivalant to "screen height * (9:16)"                     -->
+ <div style="
+        width: calc(100vh * 0.5625); 
+        max-width: 100%;
+        margin: 0 auto;">
```
Comment (gkatsev) [inline sandbox/vertical-video.html.example:22]: I think this outer div can be removed and instead the aspect ratio option be passed to the player.

File: sandbox/vertical-video.html.example [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    preload="auto"
+    class="vjs-fluid"
+    >
+    <source src="http://www.exit109.com/~dnn/clips/RW20seconds_1.mp4" type="video/mp4">
+    <p class="vjs-no-js vjs-">To view this video please enable JavaScript, and consider upgrading to a web browser that <a href="http://videojs.com/html5-video-support/" target="_blank">supports HTML5 video</a></p>
+  </video-js>
+</div>
+  <script>
+    var vid = document.getElementById('vid1');
+    var player = videojs(vid, {controlBar: {volumePanel: {inline: false}}});
```
Comment (gkatsev) [inline sandbox/vertical-video.html.example:35]:
```suggestion
    var player = videojs(vid, {
      aspectRatio: '9:16',
      controlBar: 
        {volumePanel: {inline: false}
      }
    });
```

File: sandbox/vertical-video.html.example (hunk 2 above)
Comment (RyKilleen) [reply sandbox/vertical-video.html.example:22]: Does the fluid nature of VideoJS take the screen height into account? That was not my experience with it.

Comment (gkatsev) [reply sandbox/vertical-video.html.example:22]: It's possible that a `min-height` or something like that should be added to the player div. But with the aspect ratio it should mostly do the right thing.

**Pull request closed without merging**